extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -16,10 +16,12 @@ class MercuryApiController extends WikiaController {
 	const WIKI_IMAGE_SIZE = 500;
 
 	private $mercuryApi = null;
+	private $mainPageHandler = null;
 
 	public function __construct() {
 		parent::__construct();
 		$this->mercuryApi = new MercuryApi();
+		$this->mainPageHandler = new MercuryApiMainPageHandler($this->mercuryApi);
```

Comment:
mainPageHandler should have only static methods (same as article handler and category handler)